source/core/NumericString.cpp

```diff
@@ -206,7 +206,7 @@ void ReadPercentFormatOptions(SubString *format, FormatOptions *pOptions)
 }
 //---------------------------------------------------------------------------------------------
 void GenerateFinalNumeric (const FormatOptions* , char* , Int32* , TempStackCString* , Boolean );
-void RefactorLabviewNumeric(const FormatOptions* , char* , Int32* , Int32 , Int32 );
+void RefactorLVNumeric(const FormatOptions* , char* , Int32* , Int32 , Int32, Boolean );
```

Comment:
We also have LV error codes which we must support, so I don't think we need to pretend there is no relationship to LabVIEW.   I did this rename because it wasn't capitalized correctly and just seemed too verbose.